lib/src/cli/_sources.dart

```diff
@@ -1,26 +1,26 @@
-// FILE GENERATED BY scripts/update-cli.sh - DO NOT MODIFY BY HAND
+// FILE GENERATED BY /home/runner/work/sentry-dart-plugin/sentry-dart-plugin/scripts/update-cli.sh - DO NOT MODIFY BY HAND
```

Comment:
@vaind was this intentional?

Reply:
No, I was curious if that would happen though... Was hoping it would stay relative as locally. I'll update the script